plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/ReporterType.kt

```diff
@@ -0,0 +1,11 @@
+package org.jlleitschuh.gradle.ktlint
+
+import net.swiftzer.semver.SemVer
+
+@Deprecated("Moved to org.jlleitschuh.gradle.ktlint.reporter package")
```

Comment:
:x: For the next few releases use a typealias, then we can move it.
See the section here about "managing change"
https://opencredo.com/some-uses-for-type-aliases-in-kotlin/

I think if you leave the typealias behind and move the class then everything should still compile for old users.
Not 100% certain. It would nice if you could test that to make sure I'm right.

Reply:
Nice trick. Checked and it actually works for users that will update from previous plugin versions.